web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -103,6 +126,12 @@ const MemberSelectorDialog = ({ open, selectedMembers, onClose, onSubmit }) => {
       setFilteredMembers([]);
       setDirectReportsOnly(false);
       setSelectableMembers([]);
+    } else {
+      // If the dialog is opened with initial filters, set the initial filter
+      if (initialFilter) {
+        setFilterType(initialFilter.type);
+        setFilter(selectRoles(state).find(role => role.role === 'PDL'));
```

Comment:
Does this make it so only members that are PDLs are displayed?

Reply:
This will set the default of the disabled form control, so it displays the correct default. In OP when we say "clamp" we're referring to disabling (with selected defaults) the form controls in the member select dialog. Since we only support one filter-type for #2211 I've updated this branch logic to be more explicit and look for the ROLE setting.